Gemfile

```diff
@@ -9,6 +9,8 @@ gem 'jquery-rails'
 gem 'devise'
 gem "haml-rails"
 gem "foundation-rails"
+gem 'carrierwave'
+gem 'fog-aws'
 
```

Comment:
I will pretend not to be annoyed at this alternating between the use of double quotes and the use of single quotes (but maybe pick a convention and stick with it, at least inside the Gemfile?)